ENIGMAsystem/SHELL/Audio_Systems/DirectSound/DSsystem.cpp

```diff
@@ -17,7 +17,7 @@
 
 #include "Widget_Systems/widgets_mandatory.h"  // DEBUG_MESSAGE
 
-#include "Bridges/Win32/WINDOWShandle.h" //get_window_handle()
+#include "Platforms/Win32/WINDOWSmain.h" // enigma::hWnd
```

Comment:
I said that because I didn't know at the time we had anything for this under bridges. I remembered vaguely the pr which did that never got merged, but apparently it did.